### ISSUE TITLE
Support trades from clipper

### ISSUE THREAD

**Issue by yabirgb** [rotki/rotki#3216]
## Abstract

Add support for clipper in the same way we support uniswap, balancer...

## Motivation

Clipper is constantly offering better prices for small changes using 1inch and more and more people are using it

## Specification

https://clipper.exchange/

There are subgraphs that we can use

**Comment by LefterisJP** [comment]
What the hell is clipper? :sweat_smile:

**Comment by yabirgb** [comment]
> What the hell is clipper? sweat_smile

There is a link. Is a AMM optimized for small trades

**Comment by LefterisJP** [comment]
Never heard of it before. Please let's focus on the integrations that will bring the most value to the biggest percentage of our users.

**Comment by yabirgb** [comment]
I agree that is not a priority but it has been the best path in 1inch for more than 1 month already. I got many trades from it 

![image](https://user-images.githubusercontent.com/5068010/126903593-923ed677-5a9b-4024-8acb-c3cfe4847890.png)
